ci/do_ci.sh

```diff
@@ -150,7 +150,7 @@ function run_process_test_result() {
 
 function run_ci_verify () {
   echo "verify examples..."
-  docker load < "$ENVOY_DOCKER_BUILD_DIR/docker/envoy-docker-images.tar.xz"
+  LC_ALL=C docker load < "$ENVOY_DOCKER_BUILD_DIR/docker/envoy-docker-images.tar.xz"
```

Comment:
im struggling to understand how this breaks - weve been using the updated build image for some time

Reply:
also this is not run in the build image - so this overriding the locales in the CI (ie azure) image (afaict)

Reply:
Yes, it changes locale in the CI, not build image, so that byte sequences passed to `docker load/save` are interpreted correctly.

Previously, it resulted in EOF during `docker load`when reading from pipe, perhaps it was a flake, but there was another fallout because of the UTF-8 change and I was running against the clock. Unfortunatelly, I cannot find that run in Azure Pipelines now, though.

Reply:
but docker load/save wasnt/isnt broken